helm/idol-nifi/values.yaml

```diff
@@ -217,6 +232,39 @@ nifiRegistry:
       cpu: 1000m
       memory: 1Gi
 
+nifiMetrics:
+  servicePort: 443
+  containerPort: 6443
```

Comment:
Is there any purpose to these being configurable? As far as I can see both only used within `nifi-metrics.yaml` and I can't see that the ports could clash with anything.

Reply:
If anything needs to be configurable I would argue it's the `9092` port being used on the nifi container to serve the metrics, as presumably the flow could conceivably want to do something else on that port (though this seems unlikely as you'd have to create an additional k8s Service etc if you wanted to access it).

Reply:
I have removed the configuration options for these and used the values directly in nifi-metrics.yaml.  These were configurable as I was using https://github.com/opentext-idol/idol-containers-toolkit/blob/main/helm/distributed-idol/templates/custom-metrics.yaml#L12-L13 as an example to follow.

You could argue that any/all of the ports used by the nifi container should be configurable as someone may want to use them for something else. As the other ports used by the container are not yet configurable, I think making all of them configurable could be done as a separate change